src/Helper/ProcessHelper.php

```diff
@@ -80,7 +80,10 @@ public function runCommand($command, $allowFailures = false, $callback = null, $
         $process = $builder->getProcess();
 
         $remover = function ($untrimmed) {
-            return ltrim(rtrim($untrimmed, "'"), "'");
+            $removedSingleQuotes = ltrim(rtrim($untrimmed, "'"), "'");
+            $removedDoubleuotes = ltrim(rtrim($removedSingleQuotes, '"'), '"');
```

Comment:
Its better to leave the output as it was executed. Its meanly for debugging so normally you would not see it :)